docs/build-instructions/linux.md

```diff
@@ -128,13 +132,15 @@ See also [#2082](https://github.com/atom/atom/issues/2082).
 ### /usr/bin/env: node: No such file or directory
 
 If you get this notice when attempting to `script/build`, you either do not
-have Node.js installed, or node isn't identified as Node.js on your machine.
-If it's the latter, entering `sudo ln -s /usr/bin/nodejs /usr/bin/node` into
+have Node.js installed, or node is not identified as Node.js on your machine.
+If it is the latter, entering `sudo ln -s /usr/bin/nodejs /usr/bin/node` into
```

Comment:
Is there a reason for eliminating contractions? I feel not using contractions can seem overly formal in something like build instructions.